meorg_client/docs/cli.md

```diff
@@ -0,0 +1,144 @@
+# Command-Line Usage
+
+The client can be used over the command line in a standard UNIX environment.
+
+> NOTE: The command-line utilities will only work if you have been granted API access by modelevaluation.org.
+
+## Set up credentials.
+
+Credentials to the system are stored in the user home directory `$HOME/.meorg/credentials.json`. To set up credentials follow these steps:
+
+1. Get an account on modelevaluation.org. Take note of the email and password used.
+2. Run the command `meorg initialise` on the machine where you have the client installed.
+3. Follow the prompts.
+
+The system will attempt to authenticate with modelevaluation.org, this will write the credentials file upon success. 
+
+> NOTE: This will overwrite any existing credentials.json
+
+Alternatively, you can create a credentials file at the target filepath manually in the following format:
```

Comment:
It is a bit unclear on how the user should specify a credentials file, is there an argument to `meorg initialise` that takes in a path to a `credentials.json` file?

Reply:
A credentials.json file is created using the wizard invoked by `meorg initialise`. It does not take arguments, rather, it asks for user input to supply the email/password combination to access modelevaluation.org

I've added some clarity to the docs.